ext/src/helpers/wrapped_struct.rs

```diff
@@ -18,6 +18,7 @@ impl<T: TypedData> Clone for WrappedStruct<T> {
         }
     }
 }
+impl<T: TypedData> Copy for WrappedStruct<T> {}
```

Comment:
Don't see a reason why not, under the (transparent) wrappers it's basically a `VALUE`.